contrib/buildrefactor/tests/python/pants_test/contrib/buildrefactor/test_buildozer.py

```diff
@@ -56,7 +55,7 @@ def test_custom_command(self):
     new_build_name = 'b_2'
 
     self._run_buildozer({ 'command': 'set name {}'.format(new_build_name) })
-    self.assertInFile(new_build_name, os.path.join(self.build_root, 'b/BUILD'))
+    self.assertInFile(new_build_name, '{}/b/BUILD'.format(self.build_root))
```

Comment:
Using `os.path.join` is intentional here. in fact, I missed it in the last review, it should be `os.path.join(self.build_root, 'b', 'BUILD').

Same applies for other instances.